### PR TITLE
Add extension larger 20mb for testing purposes

### PR DESCRIPTION
**Pull request by aalves08**
add extension larger than 20mb that will be used for e2e testing issue https://github.com/rancher/dashboard/issues/10639

### CODE REVIEW

**Review by jordojordo — COMMENTED** [review]
Is this going to produce the expected results with only having the sum of extension files greater than 20mb and not the imported javascript being greater than 20mb?

File: assets/index.yaml [hunk 1]
```diff
@@ -0,0 +1,229 @@
+apiVersion: v1
```
Comment (jordojordo) [inline assets/index.yaml:1]: This file should be moved to the root directory

**Review by jordojordo — CHANGES_REQUESTED** [review]

File: assets/index.yaml (hunk 1 above)
Comment (aalves08) [reply assets/index.yaml:1]: true true... It didn't update the root `index.yaml` 🤦

**Review by rak-phillip — APPROVED** [review]
LGTM - I'm not sure if I would call this clock-large, but I think this will satisfy the need for an extension large enough to disable the cache. I'm unfortunately not able to test properly via the developer load function..

**Comment by aalves08** [comment]
Output from a developer load. Bottom line is: it will work as expected (cache disabled):

Screenshot 2024-05-14 at 11 48 04
Screenshot 2024-05-14 at 11 47 06
Screenshot 2024-05-14 at 11 47 23

**Review by jordojordo — APPROVED** [review]

**Pull request merged**